blebox_uniapi/box_types.py

```diff
@@ -170,17 +169,16 @@ def get_latest_api_level(product_type: str) -> Union[dict, int]:
                         "state": "thermo/state",
                         "mode": "thermo/mode",
                         "safetySensorId": "thermo/safetyTempSensor/sensorId",
-                        "operatingState": "thermo/operatingState"
-                    }
+                        "operatingState": "thermo/operatingState",
+                    },
                 ]
-            ]
+            ],
         }
-
     },
     "saunaBox": {
         20180604: {
             # TODO: read extended state only once on startup
-            "api_path": "/api/heat/extended/state",
+            "api_path": "/api/heat/state",
```

Comment:
Endpoint doesn't provide information about maximumTemp and minimumTemp.
https://technical.blebox.eu/archives/saunaBoxAPI/